.flake8

```diff
@@ -37,11 +37,6 @@ exclude =
 
 # Completely and unconditionally ignore the following errors:
 ignore =
-  # TODO(ssbarnea): gradually address exceptions below:
-  D100  # D100 Missing docstring in public module
-  D101  # D101 Missing docstring in public class
-  D102  # D102 Missing docstring in public method
-  D103  # D103 Missing docstring in public function
   # W503 is incompatible with W504
   W504  # W504 line break after binary operator
 
```

Comment:
Looks like this section can be removed completely. Both W503 and W504 are in default_ignore.

Reply:
Is not, removing them would raise ~20 W504 errors.

Reply:
This is weird: https://github.com/PyCQA/pycodestyle/blob/1797af2/pycodestyle.py#L84

Reply:
btw it looks like it's recommended to use `extend-ignore` instead of `ignore`: https://gitlab.com/pycqa/flake8/-/issues/466#note_111487130

Reply:
@ssbarnea locally, removing it doesn't cause violations. Did you just remove the option or the whole setting? I can imaging that removing an option will redefine default ignores to an empty string which is not what we want.